sdk/iot/README.md

```diff
@@ -1,14 +1,38 @@
-# Azure IoT Clients for Embedded C
+# Embedded C for Azure IoT Clients
 
-Official Embedded C clients for Azure IoT Hub and Device Provisioning Service.
+Official Embedded C clients for Azure IoT Hub services.

```

Comment:
Fix naming, see comment further down.

Reply:
I think we should keep both Hub and Provisioning: they are two different Azure Service Offerings, can be purchased separately and have separate clients.
Otherwise you may want to delete Hub which is one of the Azure IoT services (but Azure has a lot of IoT services that are not covered by this SDK).